package.json

```diff
@@ -113,5 +113,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "volta": {
```

Comment:
Added this because I thought node was bugging out on me. It wasn't, I just was putting `ConstantNumberAssembler` into getComponentInfo instead of `ConstantNumber`. Left it in though because Volta is a tool that automatically swaps to node/yarn versions you have defined in package.json like this, so nice to have. And it'll make future upgrades of these easier